pkg/common/utils/utils.go

```diff
@@ -257,3 +259,109 @@ func GetUserFromPod(pod *v1.Pod) string {
 
 	return value
 }
+
+func GetPodAnnotationValue(pod *v1.Pod, annotationKey string) string {
+	if value, ok := pod.Annotations[annotationKey]; ok {
+		return value
+	}
+	return ""
```

Comment:
Nit: {{""}} is used as a magic value. Consider using a constant called {{none}}.

Reply:
Addressed in latest PR, thanks!

Reply:
Hi @pbacsko , revert the "" to none change, it will affect many unit tests, and will affect the PB value originally will be "".